elliott/elliottlib/cli/create_cli.py

```diff
@@ -112,7 +115,7 @@ def create_cli(ctx, runtime, errata_type, kind, impetus, date, assigned_to, mana
             et_data,
             errata_type=errata_type,
             kind=kind,
-            boilerplate_name=(impetus if impetus != "standard" else kind),
```

Comment:
we pass standard for non microshift here is incorrect, it will make advisory creation check failed https://github.com/openshift-eng/art-tools/blob/main/elliott/elliottlib/errata.py#L238 and create from default value not the boilerplate config

Reply:
I looked at impetus values, why not remove all values that are not in boilerplate?
```
boilerplate_name = (impetus if impetus not in ['standard', 'cve', 'ga', 'test'] else kind)
```

Reply:
And then whichever place we are calling create for microshift, we make sure to pass it as impetus?

Reply:
you can see the invoking from preparing releaae job. the issue here is impetus is different usage here. we did't pass sort like extras or metadata type here.

Reply:
Okay I traced how we use impetus, impetus is basically the boilerplate name. I think we should refactor and fix our use of impetus instead of adding "sort". Let's start by analyzing `elliottlib.constants.errata_valid_impetus` unused values, why do we have ga/test defined if we don't have boilerplate value for them? I would prefer removing "standard" as well, if impetus is not defined we already fall back on kind, so we don't really need an explicit value for it like "standard". Once we have removed ga and standard, we can simply pass the advisory type as impetus.

Reply:
It is misleading with those types, I would like to get this fix pr in so docs can have correct content and refactor the implementation later in a separate card

Reply:
It is misleading with those types, I would like to get this fix pr in so docs can have the correct content and refactor the implementation later in a separate card

Reply:
as it described in https://github.com/openshift-eng/art-tools/blob/main/elliott/elliottlib/cli/add_metadata_cli.py#L33 the impetus is used in an advisory's comment like https://errata.devel.redhat.com/advisory/123315#c1, not sure if it's still useful for someone, maybe we can try remove the step to see if someone complain
and for the kind it said https://github.com/openshift-eng/art-tools/blob/main/elliott/elliottlib/errata.py#L210 I think it's used for distinguished the key in boilerplates so maybe it can also removed

Reply:
@thegreyd I created [ART-8191](https://issues.redhat.com/browse/ART-8191)